e2e-polybft/property_test.go

```diff
@@ -0,0 +1,57 @@
+package e2e
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/e2e-polybft/framework"
+	"github.com/0xPolygon/polygon-edge/types"
+	"pgregory.net/rapid"
+)
+
+// property based tests enabled
+const envPropertyBaseTestEnabled = "E2E_PROPERTY_TESTS"
+
+func TestProperty_DifferentVotingPower(t *testing.T) {
+	if strings.ToLower(os.Getenv(envPropertyBaseTestEnabled)) != "true" {
+		t.Skip("Property based tests are disabled.")
+	}
+
+	t.Parallel()
+
+	const (
+		blockTime  = time.Second * 6
+		maxPremine = math.MaxUint64
+	)
+
+	rapid.Check(t, func(tt *rapid.T) {
+		var (
+			numNodes  = rapid.Uint64Range(4, 8).Draw(tt, "number of cluster nodes")
+			epochSize = rapid.OneOf(rapid.Just(4), rapid.Just(10)).Draw(tt, "Size of epoch")
```

Comment:
Nitpick (consistency-wise):
```suggestion
			epochSize = rapid.OneOf(rapid.Just(4), rapid.Just(10)).Draw(tt, "epoch size")
```

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1149/commits/47178fb01acb738db85dab15f573b2ceebc50c0a